docs/dispatch/Error_Codes.md

```diff
@@ -14,7 +14,7 @@ This page outlines some of the common errors codes that may be encountered when
 | 2025 | Install Script Failed      | Restart Discord, attempt to uninstall/reinstall the game, ensure script is correct                                                             |
 | 2029 | Build Not Found            | Completely close and re-open Discord                                                                                                           |
 | 2051 | Panic!                     | Escalate in the dev server in #dispatch                                                                                                        |
-| 2058 | Too Many API Retries       | Escalate in dev server in #dispatch                                                                                                            |
+| 2058 | Too Many API Retries       | Escalate in the Discord-Developers server in #dispatch                                                                                                            |
```

Comment:
If we want to change this we should at least link the vanity URL for the server. https://discord.gg/discord-developers

Reply:
hmm good idea